<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-NAT0VEE" failures="2" tests="4" name="PetTest" time="13.547" errors="0" timestamp="2025-03-07T15:09:21 BRT">
  <testcase classname="api.test.UserTest2" name="testPostUser" time="6.181"/>
  <testcase classname="api.test.UserTest2" name="testGetUserByName" time="0.817">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTest2.testGetUserByName(UserTest2.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserByName -->
  <testcase classname="api.test.UserTest2" name="testUpdateUserByName" time="1.578">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTest2.testUpdateUserByName(UserTest2.java:82)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.test.UserTest2" name="testDeleteUserByName" time="0.761"/>
</testsuite> <!-- PetTest -->
